pygeo/constraints/baseConstraint.py

```diff
@@ -58,13 +58,23 @@ def getVarNames(self):
         """
         return self.DVGeo.getVarNames()
 
-    def addConstraintsPyOpt(self, optProb):
+    def addConstraintsPyOpt(self, optProb, exclude_wrt=None):
```

Comment:
Also need docstrings

Reply:
added. let me know if you want any other changes here